lib/modules/manager/asdf/extract.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { extractPackageFile } from '.';
+
+describe('modules/manager/asdf/extract', () => {
+  describe('extractPackageFile()', () => {
+    it('returns a result', () => {
+      const res = extractPackageFile('nodejs 16.16.0\n');
+      expect(res.deps).toEqual([
+        {
+          currentValue: 'nodejs 16.16.0',
+          datasource: 'github-tags',
+          depName: 'node',
+          packageName: 'nodejs/node',
+          versioning: 'node',
+        },
+      ]);
+    });
```

Comment:
```suggestion
      expect(res).toEqual({ deps: [
        {
          currentValue: 'nodejs 16.16.0',
          datasource: 'github-tags',
          depName: 'node',
          packageName: 'nodejs/node',
          versioning: 'node',
        },
      ]});
    });
```

res is nullable, so use this variant for test validation